src/index.js

```diff
@@ -1,19 +1,20 @@
 import express from 'express';
-import { serve, setup } from 'swagger-ui-express';
-import swaggerJsdoc from 'swagger-jsdoc';
-import swaggerDefinition from './swagger';
+import bodyParser from 'body-parser';
+import registerApiDocs from './swagger';
+import router from './routes/auth';
```

Comment:
The import name here should be `auth` and not router. See the import and export convention in airbnb documentation